assets/queries/k8s/default_namespace/query.rego

```diff
@@ -0,0 +1,30 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    
+    not document.metadata.namespace
+
+	result := {
+                "documentId": 		  input.document[i].id,
+                "issueType":		  "MissingAttribute",
+                "searchKey": 	      "metadata",
+                "keyExpectedValue":   sprintf("Namespace is set in document[%d]",[i]),
+                "keyActualValue": 	  sprintf("Namespace is not set in document[%d]",[i])
+              }
+}
+
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    
+    document.metadata.namespace == "default"
+
+	result := {
+                "documentId": 		  input.document[i].id,
+                "issueType":		  "IncorrectValue",
+                "searchKey": 	      "metadata.namespace",
```

Comment:
Add metadata information in order to have a more specific searchKey
"searchKey": sprintf("metadata.name=%s.metadata.namespace, [metadata.name])